src/controllers/connection/helpers.ts

```diff
@@ -3,10 +3,11 @@ import { ILogger } from '../../logger.js'
 import { ConnectionRow } from '../../models/db/types.js'
 
 export const checkDb = (rows: ConnectionRow[], initialPinAttemptsRemaining: number | null, logger: ILogger) => {
+  logger.trace('check(): called, %j', { rows, initialPinAttemptsRemaining, logger })
   const [connectionCheck] = rows
 
   if (connectionCheck.status === 'verified_us' || connectionCheck.status === 'verified_both') {
-    logger.debug('Pin has been verified.')
+    logger.info('Pin has been verified. [%s]', connectionCheck.status)
```

Comment:
so everything in this file should be lowercase?

Reply:
it's not part of ACs, i think we are getting into too much detail.